moveit_core/robot_state/src/cartesian_interpolator.cpp

```diff
@@ -159,12 +265,12 @@ CartesianInterpolator::Percentage CartesianInterpolator::computeCartesianPath(
     Eigen::Isometry3d pose(start_quaternion.slerp(percentage, target_quaternion));
     pose.translation() = percentage * rotated_target.translation() + (1 - percentage) * start_pose.translation();
 
-    // Explicitly use a single IK attempt only: We want a smooth trajectory.
+    // Explicitly use a single IK attempt only: We want a smooth path.
```

Comment:
Yeah, this is a bit misleading, and a confusing API too, since the single call is apparently achieved by setting a timeout of 0.0.
Even if a single seed is used (the current state), since this is calling a pose-IK solver underneath (the configured plugin), you can still have large joint-space reconfigurations, depending on the specific solver implementation.
I updated the comment but wouldn't worry much more about this, since it will soon be deprecated.